ts/nni_manager/training_service/local/localTrainingService.ts

```diff
@@ -253,7 +253,16 @@ class LocalTrainingService implements TrainingService {
 
             return Promise.resolve();
         }
-        tkill(trialJob.pid, 'SIGKILL');
+        tkill(trialJob.pid, 'SIGTERM');
+        let waitingTime = 0;
+        while(await isAlive(trialJob.pid)) {    
+            if (waitingTime > 4999) {
+                tkill(trialJob.pid, 'SIGKILL');
```

Comment:
how about add `break` here, in case pid is not terminated even with sigkill?

Reply:
`SIGKILL` can not kill process is almost impossible, if it is happened, I think there may have a serious system error. I will add `catch error` and `break` in this case.